contrib/format-esri/pom.xml

```diff
@@ -18,7 +18,8 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please revert this change. `maven-release-plugin` automatically reformats the header line when `release:prepare` is performed.

Reply:
Reverted first line spacing.  Just as a side note, I ran Drill's code formatting macro in Intellij which did this.  We might want to think about updating that so that this line is formatted correctly in future PRs, but that is out of scope for this PR.

Reply:
I agree that our templates for both IDEA and Eclipse are a little bit outdated, but I believe they will be updated when we will add more checkstyle rules and reformat the project for them ([DRILL-7352](https://issues.apache.org/jira/browse/DRILL-7352)).

But for now, sorry for asking to do additional changes.

Reply:
No worries.  This kind of stuff is good for Drill. :-)